test/services/EscalationLogic.spec.ts

```diff
@@ -0,0 +1,60 @@
+import 'mocha';
+import chai from 'chai';
+import spies from 'chai-spies';
+import mongoose from 'mongoose';
+import message from '@test/data/message';
+import MessageService from '@/services/MessageService';
+import DeviceService from '@/services/DeviceService';
+import device from '@test/data/device';
+import Utils from '@/utils';
+
+// Add extensions to chai
+chai.use(spies);
+const expect = chai.expect;
+
+describe('EscalationLogic.escalate', () => {
+
+  // Instantiate the service
+  const messageService = new MessageService();
+
+  before('should establish a database connection.', (done) => {
+    // connect to database
+    const db = mongoose.connection;
+    db.on('error', console.error.bind(console, 'connection error:'));
+    db.once('open', done);
+    mongoose.connect(`mongodb://${process.env.MONGO_HOST || 'localhost'}/notification-service-test`);
```

Comment:
The url should probably be moved to an own test config file. I would prefer this in `/test/config.ts